testing/test_assertion.py

```diff
@@ -1435,6 +1435,61 @@ def test_many_lines():
         result = pytester.runpytest()
         result.stdout.fnmatch_lines(["* 6*"])
 
+    @pytest.mark.parametrize(
+        ["truncation_mode", "truncation_limit", "expected_lines_hidden"],
+        (
+            ("lines", 3, 3),
+            ("lines", 4, 0),
+            ("chars", 8, 6),
+            ("chars", 9, 0),
```

Comment:
Let's add a test case for no truncation at all.

Reply:
Is that a test case, when the limits are too high to perform any truncation? If so, that is done in test-cases in lines `1442` and `1444`. 

Or is it something completely different and more applicable to the case stated in your message [here](https://github.com/pytest-dev/pytest/pull/12766#discussion_r1761602377)?

Reply:
Ahh sorry, I should have been clearer: I meant a case where the user explicitly sets both options to `0`, and then the output is not truncated at all.